lib/ansible/modules/database/postgresql/postgresql_schema.py

```diff
@@ -231,7 +232,7 @@ def main():
             cursor_factory=psycopg2.extras.DictCursor)
     except Exception:
         e = get_exception()
-        module.fail_json(msg="unable to connect to database: %s" %(text, str(e)))
+        module.fail_json(msg="unable to connect to database: {0}".format(str(e)), exception=traceback.format_exc())
```

Comment:
format is python2.6+  So keep using %.

If you want to you can also switch from str(e) to to_native(e)  where you to_native by doing from ansible.module_utils._text import to_native.

(to_native will be more resistant to tracebacks than str).